docusaurus/docs/tutorials/build-a-panel-plugin.md

```diff
@@ -156,15 +152,13 @@ You're almost done. You've added a new option and a corresponding control to cha
 
 1. To convert option value to the colors used by the current theme, add the following statement right before the `return` statement in `SimplePanel.tsx`.
 
-   **`src/components/SimplePanel.tsx`**
-
-   ```ts
+   ```ts title="src/components/SimplePanel.tsx"
    let color = theme.visualization.getColorByName(options.color);
    ```
 
 1. Configure the circle to use the color.
 
-   ```ts
+   ```tsx title="src/components/SimplePanel.tsx"
```

Comment:
just below again we've got a random

> note

which could be a proper one

Reply:
Yep, well spotted again! I've updated it to use an info admonition. 👍